src/mlpack/methods/cf/cf_impl.hpp

```diff
@@ -22,7 +22,7 @@
 namespace mlpack {
 namespace cf {
 
-// Apply the factorizer when a coordinate list is used.
+// Apply QuicSVD factorizer.
```

Comment:
misplaced comment?

Reply:
Hey. Actually it is a new function for handling `QuicSVD` enabled using `UseQuicSVD` parameter in `FactorizerTraits`. The original function with the initial comment is defined beneath that.

Reply:
I think the function here is the original one and the function beneath is the new function for `QuicSVD`.

Reply:
oops. Thanks a lot for catching that. Must have messed it up when transferring from local copy :D .